src/main/java/seedu/address/model/transaction/DescriptionContainsKeywordsPredicate.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.model.transaction;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.util.StringUtil;
+
+/**
+ * Tests that a {@code Transaction}'s {@code Description} matches any of the keywords given.
+ */
+public class DescriptionContainsKeywordsPredicate implements Predicate<Transaction> {
```

Comment:
Is this for the find/filter command?

Reply:
This is part of the WalletDeleteCommandTest, in order to check if the object deleted is actually deleted.

Reply:
Idts though it looks like it's for the find command (like the People-equivalent NameContainsKeywordsPredicate is used in PeopleFind command to filter names containing the keywords).

But i think it's fine HAHA you can leave it, one less thing to implement when you do the find command next time